.github/ISSUE_TEMPLATE.md

```diff
@@ -19,7 +19,7 @@ you can.
 6. What did you expect to happen?
 
 7. Please provide your cluster manifest. Execute
-  `kops get --name my.example.com -oyaml` to display your cluster manifest.
+  `kops get --name my.example.com -o yaml` to display your cluster manifest.
```

Comment:
does that not work?  I assumed it did

Reply:
Sorry ! I think it's more clear, if it's not necessary, i'll remove the commit.